### DISCUSSION TITLE
MSFS 2024

### DISCUSSION THREAD

**Discussion by SOFTWAREKOCH** [mpaperno/MSFSTouchPortalPlugin#67] [Q&A] [answered]
Hello,
i am using Touch Portal in combination wirh XPLANE 12.
Will Max´s plugin work for MSFS 2024 ?

Brgds Dietmar

**Comment by mpaperno** [comment]
Hi Dietmar,

It should! In theory even the WASM module should keep working.

But I'm not in any MSFS2024 testing groups, so I have no way to be sure until it is actually released. Assuming it will be available to the XBox store Game Pass subscribers, like MSFS2020, then at least I'll be able to test it right away.  If I have to buy it... then maybe not so quickly... :)

Cheers,
-Max

**Reply by SOFTWAREKOCH** [reply]
Hi Max,

a really quick respone from your side.
i was not flying on MSFS since years but now i will give it a try beside XPLANE.
I will buy the Deluxe version an then try your plugin. I will give you feedback then on github.
Would be nice to have one Touch solution for XpLANE12 and MSFS2024.  Lets hope the best.

Thx in advance
Brgds Dietmar

**Reply by mpaperno** [reply]
Thanks, feedback is always good!

I just checked their "2024" site and it does say it will be available to Game Pass on release day, Nov. 14. So hopefully we'll know soon. Of course if it is anything like the first releases of 2020, then bugs can be expected (followed by updates hopefully).  If you want to try it out before fully buying, Game Pass may be an option (maybe even free or very cheap for first month).

I'll post here once I have a chance to try it out.  🤞🏼 

Best,
-Max

**Reply by SOFTWAREKOCH** [reply]
Hi Max,
i bought the MSFS 2024 Premium Deluxe in the Microsoft Store via XBOX.
So i would be ready to test :-)

The first thing you wll have to change in your code may be the Path.
In the MSFS2024 for User "KOCH" it is :
C:\Users\KOCH\AppData\Local\Packages\Microsoft.Limitless_8wekyb3d8bbwe\LocalCache\UserCfg.opt
Brgds Dietmar

**Reply by mpaperno** [reply]
Hi Dietmar,

Hope you're enjoying 2024!  Seems it has a long way to go before it really usable, but a good start.

I can confirm the plugin works with MSFS 2024 as expected.  Including the optional `WASimModule` add-on.

> The first thing you wll have to change in your code may be the Path.

I'm not sure what you mean here...   the plugin doesn't make use of your `UserCfg` file.

Cheers,
-Max

**Comment by mpaperno** [comment, marked as answer]
So far plugin seems to work fine with MSFS2024, including the `WASimModule` add-on (if you can find your Community folder to stick it into).

Please let me know if any issues, though do keep in mind at this point MSFS 2024 has a lot of bugs that still need to be worked out (especially with 3rd party models).

I'm especially interested in any general failures like plugin or WASM module crashing or causing reproducible problems with the sim.

**Comment by SOFTWAREKOCH** [comment]
Hi Max,

 

just installed the plugin with success and copied the WASM into the community folder.

When importing the MFS Start Page, i get a MessageBox telling me 3 pieces are missing ( see below)

 

Brgds Dietmar

**Reply by mpaperno** [reply]
Hi, your attachments didn't come through with the e-mail. 

I'm assuming the message is Touch Portal telling you some plugin(s) is/are "missing" or "required."  This can usually be ignored, assuming the plugin is installed.  

In TP v4 I've also seen it get confused and call something in the page a "plugin" when it actually isn't.  For the "Started Page" the MSFS plugin should be the only one actually required.  (Make sure you get the started page for MSFS version of plugin, not FSX version.)

For any more issues with Touch Portal or demo pages please feel free to start a new discussion or file an issue here. (Or contact on [Discord](https://discord.gg/6nHKnsWkB7).)  Thanks!

-Max

**Comment by SOFTWAREKOCH** [comment]
Hi Max,

 

i will start a discussion on Github and put the pic which did not went through there, that you can see it.

 

Thx and Brgds

Dietmaer
